Packs/Infoblox/Integrations/Infoblox/Infoblox.py

```diff
@@ -346,7 +348,13 @@ def get_ip_command(client: Client, args: Dict[str, str]) -> Tuple[str, Dict, Dic
         Outputs
     """
     ip = args.get('ip')
-    raw_response = client.get_ip(ip)
+    try:
+        raw_response = client.get_ip(ip)
+    except DemistoException as e:
+        if e.message and NO_INDICATORS_FOUND in e.message:
```

Comment:
is there any other way to check for it? a status code maybe?

Reply:
it returns without status_code but just exception with message